### ISSUE TITLE
I Train steps=10, and add -map parameter, will not return something, The main process has been stuck

### ISSUE THREAD

**Issue by liudonggang** [AlexeyAB/darknet#5924]
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 161 Avg (IOU: 0.270259, GIOU: 0.165639), Class: 0.359618, Obj: 0.541298, No Obj: 0.522445, .5R: 0.000000, .75R: 0.000000, count: 4, class_loss = 675.817139, iou_loss = 0.344360, total_loss = 676.161499
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 139 Avg (IOU: 0.000000, GIOU: 0.000000), Class: 0.000000, Obj: 0.000000, No Obj: 0.492967, .5R: 0.000000, .75R: 0.000000, count: 1, class_loss = 9534.585938, iou_loss = 0.000000, total_loss = 9534.585938
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 150 Avg (IOU: 0.290262, GIOU: 0.152819), Class: 0.560282, Obj: 0.502072, No Obj: 0.512562, .5R: 0.066667, .75R: 0.000000, count: 15, class_loss = 2639.446777, iou_loss = 11.469727, total_loss = 2650.916504
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 161 Avg (IOU: 0.177288, GIOU: 0.135683), Class: 0.625241, Obj: 0.506458, No Obj: 0.522290, .5R: 0.000000, .75R: 0.000000, count: 2, class_loss = 671.481567, iou_loss = 0.034851, total_loss = 671.516418
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 139 Avg (IOU: 0.317648, GIOU: 0.268578), Class: 0.473974, Obj: 0.487153, No Obj: 0.492383, .5R: 0.050000, .75R: 0.000000, count: 20, class_loss = 9574.066406, iou_loss = 186.638672, total_loss = 9760.705078
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 150 Avg (IOU: 0.322617, GIOU: 0.242138), Class: 0.507876, Obj: 0.487669, No Obj: 0.512871, .5R: 0.200000, .75R: 0.000000, count: 10, class_loss = 2647.350342, iou_loss = 14.312500, total_loss = 2661.662842
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 161 Avg (IOU: 0.337057, GIOU: 0.272163), Class: 0.491761, Obj: 0.546346, No Obj: 0.524024, .5R: 0.142857, .75R: 0.000000, count: 7, class_loss = 685.374329, iou_loss = 0.563293, total_loss = 685.937622
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 139 Avg (IOU: 0.354102, GIOU: 0.220375), Class: 0.516804, Obj: 0.493399, No Obj: 0.491737, .5R: 0.166667, .75R: 0.000000, count: 6, class_loss = 9548.716797, iou_loss = 65.708984, total_loss = 9614.425781
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 150 Avg (IOU: 0.522310, GIOU: 0.509097), Class: 0.541810, Obj: 0.403823, No Obj: 0.512889, .5R: 0.500000, .75R: 0.000000, count: 2, class_loss = 2649.270996, iou_loss = 3.834961, total_loss = 2653.105957
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 161 Avg (IOU: 0.372038, GIOU: 0.176378), Class: 0.574405, Obj: 0.493636, No Obj: 0.524564, .5R: 0.500000, .75R: 0.000000, count: 2, class_loss = 679.599548, iou_loss = 0.232239, total_loss = 679.831787
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 139 Avg (IOU: 0.451047, GIOU: 0.396336), Class: 0.470109, Obj: 0.499812, No Obj: 0.492084, .5R: 0.400000, .75R: 0.000000, count: 5, class_loss = 9545.875000, iou_loss = 37.240234, total_loss = 9583.115234
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 150 Avg (IOU: 0.358202, GIOU: 0.273817), Class: 0.530598, Obj: 0.495160, No Obj: 0.512779, .5R: 0.166667, .75R: 0.000000, count: 12, class_loss = 2658.626465, iou_loss = 22.310791, total_loss = 2680.937256
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 161 Avg (IOU: 0.233748, GIOU: 0.063069), Class: 0.507617, Obj: 0.470578, No Obj: 0.525616, .5R: 0.000000, .75R: 0.000000, count: 3, class_loss = 682.366943, iou_loss = 0.185486, total_loss = 682.552429

 Tensor Cores are disabled until the first 3000 iterations are reached.
 (next mAP calculation at 1000 iterations)
 10: 4304.704590, 4303.606445 avg loss, 0.000000 rate, 12.514364 seconds, 640 images, 0.029122 hours left
Resizing to initial size: 608 x 608  try to allocate additional workspace_size = 59.74 MB
 CUDA allocate done!

 calculation mAP (mean average precision)...
4

**Comment by AlexeyAB** [comment]
* It can't calculate mAP at the iteration=10  unless you commented burn_in= param in cfg-file and  deviate from the manual: https://github.com/AlexeyAB/darknet#how-to-train-to-detect-your-custom-objects



* show screenshot with such information
```
./darknet detector test cfg/coco.data cfg/yolov4.cfg yolov4.weights data/dog.jpg
 CUDA-version: 10000 (10000), cuDNN: 7.4.2, CUDNN_HALF=1, GPU count: 1
 CUDNN_HALF=1
 OpenCV version: 4.2.0
 0 : compute_capability = 750, cudnn_half = 1, GPU: GeForce RTX 2070
net.optimized_memory = 0
mini_batch = 1, batch = 8, time_steps = 1, train = 0
   layer   filters  size/strd(dil)      input                output
   0 conv     32       3 x 3/ 1    608 x 608 x   3 ->  608 x 608 x  32 0.639 BF
```
